bundles/org.openhab.binding.samsungtv/README.md

```diff
@@ -89,6 +91,7 @@ TVs support the following channels:
 | artJson             | String   | RW         | Send/receive commands from the TV art websocket Channel                                                 |
 | artBrightness       | Dimmer   | RW         | ArtMode Brightness                                                                                      |
 | artColorTemperature | Number   | RW         | ArtMode Color temperature Minnimum value is -5 and maximum 5                                            |
+| artOrientation      | Switch   | RW         | TV orientation, Landscape (OFF) or Portrait (ON)                                                        |
```

Comment:
Some mixup i guess:

line 94: Landscape (OFF)
line 313: OFF for Landscape
line 319: ON="Landscape"

Reply:
You are right, must have had a brain fart. fixed.